packages/dashboard/src/features/database/helpers.ts

```
@@ -6,6 +6,69 @@ export const DEFAULT_DATABASE_SCHEMA = 'public' as const;
+  }
+  return key;
+}
+
+/**
+ * Encodes a row's full primary-key tuple into a stable string usable as a React
+ * grid row key. Two rows with the same key tuple encode identically (same identity).
+ */
+export function encodeRecordKey(
+  row: Record<string, unknown>,
+  primaryKeyColumns: string[]
+): string {
```

ESLint

[error] Replace `⏎··row:·Record<string,·unknown>,⏎··primaryKeyColumns:·string[]⏎` with `row:·Record<string,·unknown>,·primaryKeyColumns:·string[]`

(prettier/prettier)